src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -234,7 +238,60 @@ def github_comment_workflow(pullrequest_id: int, project_id: int):
         cache.delete(cache_key)
 
         if ISSUE_LOCKED_ERROR_MESSAGE in e.json.get("message", ""):
+            metrics.incr("github_pr_comment.issue_locked_error")
+            return
+
+        elif RATE_LIMITED_MESSAGE in e.json.get("message", ""):
+            metrics.incr("github_pr_comment.rate_limited_error")
```

Comment:
Can we also track gh rate limits from the header regardless of if we're hitting it or not?